app/views/taxonomy/_groups.html.haml

```diff
@@ -0,0 +1,10 @@
+.taxonomy-section
```

Comment:
rename this file to  `_taxonomies` and remove `_categories`.

and rename `@groups` to `@taxonomies`